src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -50,6 +46,7 @@
 public class SystemConfig {
 
     private static final Logger logger = Logger.getLogger(SystemConfig.class.getCanonicalName());
+    private static final Config config = ConfigProvider.getConfig();
```

Comment:
Since JVMSettings calls this internally in the lookup/lookupOptional calls, is this needed here?

Reply:
Good catch! Removed with [4b60983](https://github.com/IQSS/dataverse/pull/8824/commits/4b60983e360b3ee4b5a50535b769852fc9ea67ef)